Topic_3/shoppingCart/src/main/java/com/mycompany/app/shoppingCart/ShoppingCartServiceImp.java

```diff
@@ -0,0 +1,41 @@
+package com.mycompany.app.shoppingCart;
+
+import java.util.ArrayList;
+
+public class ShoppingCartServiceImp implements ShoppingCartService {
+	
+	
+	private ArrayList<Product> shoppingList;

```

Comment:
This must be an attribute from a ShoppingCartService. This is not a responsibility from the service.